storage/internal/benchmarks/README.md

```diff
@@ -32,7 +32,12 @@ This runs 1000 iterations on 512kib to 2Gib files in the background, sending out
 | -force_garbage_collection | whether to force garbage collection <br> before every write or read benchmark |  `true` or `false` (present/not present) | `false` |
 | -timeout | timeout (maximum time running benchmarks) <br> the program may run for longer while it finishes running processes | any [time.Duration](https://pkg.go.dev/time#Duration) | `1h` |
 | -timeout_per_op | timeout on a single upload or download | any [time.Duration](https://pkg.go.dev/time#Duration) | `5m` |
-| -workload | `1` will run a w1r3 (write 1 read 3) benchmark <br> `6` will run a benchmark uploading and downloading (once each) <br> a single directory with `-directory_num_objects` number of files (no subdirectories)  | `1` or `6` | `1` |
-| -directory_num_objects | total number of objects in a directory (directory will only contain files, <br> no subdirectories); only applies to workload 6 | any positive integer | `1000` |
+| -workload | `1` will run a w1r3 (write 1 read 3) benchmark <br> `6` will run a benchmark uploading and downloading (once each) <br> a single directory with `-directory_num_objects` number of files (no subdirectories) <br> `9`** will run a benchmark that does continuous reads on a directory with `directory_num_objects` | `1`, `6`, `9` | `1` |
+| -directory_num_objects | total number of objects in a directory (directory will only contain files, <br> no subdirectories); only applies to workload 6 and 9 | any positive integer | `1000` |
 
 \* required values
+
+\*\* Note that this workload is experimental and will not work under certain conditions. Here's a non-comprehensive list of notes on workload 9:
+ - output type must be `cloud-monitoring`
+ - it continues reading until the timeout is reached - the workload will always fail with deadline exceeded
```

Comment:
Maybe we can add a timer so it can still exit zero once the timeout is reached?

Reply:
I made a small change that will cause it to exit zero instead (returning nil instead of the error from ctx.done)